PriceKataCalc/Program.cs

```diff
@@ -7,29 +7,45 @@ namespace Kata
     class priceCalc
     {
         public static Dictionary<int, Discount> keysDiscount = new();
+               public static void Main()
+        {
+            Discount myDiscount, myUPCDiscount;
+            GenerateDiscounts(out myDiscount, out myUPCDiscount);
+            keysDiscount.Add(12345, myUPCDiscount);
+            List<string> Curr = GetCurrencies();
```

Comment:
Is there another better way to represent currencies other than strings ?

Reply:
i considered using an Enum